gamatrix-gog.py

```diff
@@ -228,24 +245,20 @@ def build_config(args):
     if "db_path" not in config:
         config["db_path"] = "."
 
-    config["all_games"] = False
-    if args.all_games:
-        config["all_games"] = True
+    config["all_games"] = args.get("--all-games", False)
 
-    config["include_single_player"] = False
-    if args.include_single_player:
-        config["include_single_player"] = True
+    config["include_single_player"] = args.get("--include-single-player", False)
 
-    if args.server:
+    if args.get("--server", True):  # Note that the --server opt is False unless present
```

Comment:
`--server` is always set if I'm understanding the comment, so why set the default value?

Reply:
Ah crap! This should read `.get("--server", False)` based on the comment, and also based on the behaviour we want. I'll change it.